docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -45,25 +45,23 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.services** | No | Contains all services that the Application provides. |
 | **spec.services.id** | Yes | Identifies the service that the Application provides. |
 | **spec.services.identifier** | No | Represents an additional identifier unique in the Application scope. Allows the external system to provide its own identifier. |
-| **spec.services.name** | No | Represents a unique name of the service. Used for proxying in Central Application Gateway. |
-| **spec.services.displayName** | Yes | Specifies a human-readable name of the Application service. Parameter provided by Application Registry, do not edit. |
-| **spec.services.description** | No | Provides a short, human-readable description of the service offered by the Application. Parameter provided by Application Registry, do not edit. |
-| **spec.services.longDescription** | No | Provides a longer, human-readable description of the service offered by the Application. Parameter provided by Application Registry, do not edit. |
-| **spec.services.providerDisplayName** | Yes | Specifies a human-readable name of the Application service provider. Parameter provided by Application Registry, do not edit. |
-| **spec.services.tags** | No | Specifies additional tags used for better documentation of the available APIs. Parameter provided by Application Registry, do not edit. |
-| **spec.services.labels** | No | Specifies additional labels for the service offered by the Application. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries** | Yes | Contains the information about the APIs and events that the service offered by the Application provides. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries.type** | Yes | Specifies the entry type: API or event. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. Internal address resolvable only within the cluster. This field is required for the API entry type. Parameter provided by Application Registry, do not edit. |
+| **spec.services.name** | No | Represents a unique name of the service. Used for proxying in Application Gateway. |
+| **spec.services.providerDisplayName** | Yes | Specifies a human-readable name of the Application service provider. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
```

Comment:
```suggestion
| **spec.services.providerDisplayName** | Yes | Specifies a human-readable name of the Application service provider. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

Reply:
I think this is mostly preferential. I prefer to link the smallest meaningful bit, and I feel like `standalone` is just part of it. I'm going to leave it as is :)

Reply:
I replied before I noticed that you actually removed duplication here. Thanks!